scripts/update_setup_py.sh

```diff
@@ -125,9 +49,9 @@ fi
 # Create PR description for use by cleanup-python-code jenkins job
 echo -e "[ARCHBOM-1772](https://openedx.atlassian.net/browse/ARCHBOM-1772)
  Update setup.py to use constraint files when generating requirements files for packaging and distribution.
- PR generated automatically with Jenkins job cleanup-python-code. " > .git/cleanup-python-code-description;
-echo -e "\nResult of running \`python setup.py bdist_wheel\` before applying fix (in .egg-info/requires.txt)\: \n" >> .git/cleanup-python-code-description;
-cat ./update-setup-tmp/old_requires.txt >> .git/cleanup-python-code-description;
-echo -e "\nResult of running \`python setup.py bdist_wheel\` after applying fix (in .egg-info/requires.txt)\: \n" >> .git/cleanup-python-code-description;
-cat "$(pwd)/$wheel_dir/requires.txt" >> .git/cleanup-python-code-description;
+ PR generated automatically with Jenkins job cleanup-python-code. " > .git/cleanup-python-code-description
```

Comment:
Yeah, it's weird, isn't it? I think the idea is to allow running a "commit all changes" script without having to worry about excluding the temporary file, or having to manage a file in a temp dir somewhere else. I've noticed that `git gui` actually stashes a backup of partially written commit messages in `.git/GITGUI_BCK` so it might be a more widespread approach.

Reply:
Since my only concern is unchanged by this pull request, we can approve it.